cmd/process-agent/collector.go

```diff
@@ -206,13 +208,14 @@ func (l *Collector) run(exit chan struct{}) error {
 				statsd.Client.Gauge("datadog.process.agent", 1, tags, 1) //nolint:errcheck
 			case <-queueSizeTicker.C:
 				updateQueueBytes(processResults.Weight(), podResults.Weight())
+				updateQueueSize(rtProcessResults.Len(), podResults.Len())
```

Comment:
`updateQueueSize()` updates `infoProcessQueueSize` with the first parameter. 
https://github.com/DataDog/datadog-agent/blob/82eab329be24547ed4518e44bac8b2262980e127/cmd/process-agent/info.go#L145-L150

We use it to populate `Process Queue length` when running `process-agent --info`. 

I think that we're populating this value with the length of the `rtProcessResults` and then overriding it with the lenght of the `processResults` queue.

Should we extend `udpateQueueBytes` and `updateQueueSize` to accept a third parameter regarding the realtime queue and also add the relevant info here?
https://github.com/DataDog/datadog-agent/blob/82eab329be24547ed4518e44bac8b2262980e127/cmd/process-agent/info.go#L48-L55